README.md

```diff
@@ -8,7 +8,7 @@ This repo is a template to create a new task for the OpenProblems v2. This repo
 ## Create a repository from this template
 
 > [!IMPORTANT] 
-> Before creating a new repository, make sure you are part of the openProblems task team. This will be done when you create an issue for the task and you got the go ahead to create the task.
+> Before creating a new repository, make sure you are part of the openProblems task team. This will be done when you create an issue for the task and you get the go ahead to create the task.
```

Comment:
```suggestion
> Before creating a new repository, make sure you are part of the OpenProblems task team. This will be done when you create an issue for the task and you get the go ahead to create the task.